index.rst

```diff
@@ -6,24 +6,474 @@
 
 .. TODO: Delete the note below before merging new content to the main branch.
 
-.. note::
-
-   **This technote is a work-in-progress.**
+.. Make in-text citations with: :cite:`bibkey`.
+.. Uncomment to use citations
+.. .. rubric:: References
+.. 
+.. .. bibliography:: local.bib lsstbib/books.bib lsstbib/lsst.bib lsstbib/lsst-dm.bib lsstbib/refs.bib lsstbib/refs_ads.bib
+..    :style: lsst_aa
 
 Abstract
 ========
 
-Principles, guidelines, conventions, formatting style, and workflow for tutorials developed by the Community Engagement Team (CET).
+A guideline for anyone creating or updating tutorials for the Rubin science community.
```

Comment:
Use Vera C. Rubin Observatory the first time it is used in the text (per the naming convention user guide: https://project.lsst.org/documents/name-use-guide)

Reply:
Good catch, made a branch for tickets/PREOPS-1332, from PREOPS-1332.  Also added Vera C. Rubin